portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro

```diff
@@ -162,6 +162,12 @@ definition {
 		ACSettings.goToSettingsNavigation(settingsNavigation = "definitions");
 	}
 
+	macro goToDefinitionsAttributesTabs {
+		var key_tabName = "${tabName}";
+
+		Click(locator1 = "ACNavigation#DEFINITIONS_ATRIBUTES_TABS");
+	}
```

Comment:
Hi @91nilo, the switchTab macro basically has the same function as this one, to work in your test, make the change I suggested for you in the other comment. [Click here](https://github.com/alexcandidos/liferay-portal/pull/505/files#r683682399)